### DISCUSSION TITLE
How to use addon setting values in userstyles?

### DISCUSSION THREAD

**Discussion by Secret-chest** [ScratchAddons/ScratchAddons#2411] [Help] [answered]
cc @WorldLanguages @mxmou
How can I use a string addon setting value in my userstyle? Do I need a userscript for this?

**Comment by mxmou** [comment, marked as answer]
Scratch Addons automatically sets CSS variables with names like this: `--addonName-settingName`.

**Reply by Secret-chest** [reply]
Ok thanks!
